website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -93,6 +94,22 @@ Notifications to Slack will contain the following information:
 * The time the run was created
 * The event that triggered the notification, and the time it occurred
 
+### Microsoft Teams
+
+Notifications to Microsoft Teams will contain the following information:
```

Comment:
```suggestion
Notifications to Microsoft Teams contain the following information:
```
```suggestion
Notifications to Microsoft Teams will contain the following information:
```

Reply:
Duplicate of https://github.com/hashicorp/terraform-docs-common/pull/19#discussion_r889136942